src/main/java/frc/robot/subsystems/ArmJointSystem.java

```diff
@@ -59,7 +56,7 @@ public double getPositionDegrees(){
     }
 
     public void moveToPosition(double positionDegrees){
-        pidController.setReference(positionDegrees, SparkBase.ControlType.kPosition);
+        pidController.setReference(positionDegrees, SparkBase.ControlType.kPosition, ClosedLoopSlot.kSlot0, Math.cos(Math.toRadians(getPositionDegrees())) * RobotMap.ARM_JOINT_KF, SparkClosedLoopController.ArbFFUnits.kPercentOut);
```

Comment:
Separate the FF calculation into a different line so that the line is more easily readable